changelog/6340.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed :meth:`sunpy.map.GenericMap.rotate` incorrectly transposing twice.
```

Comment:
```suggestion
Fixed an erroneous transpose of the image data by :meth:`sunpy.map.GenericMap.rotate`.
```
Not only does "transposing twice" not really accurately describe the bug, it reads silly since transposing twice would be the same as a no-op.

Reply:
Fair point!